### PR TITLE
[ipmi] Update the ipmi template for collectd

### PR DESCRIPTION
**Pull request by elfiesmelfie**
* Update README with new options
* Add ipmi to molecule/default

### CODE REVIEW

File: templates/ipmi.conf.j2 [hunk 1]
```diff
@@ -7,11 +7,57 @@ LoadPlugin ipmi
 {% endif %}
 
 <Plugin ipmi>
-  {% for sensor in collectd_plugin_ipmi_sensors %}
-  Sensor "{{ sensor }}"
+  {% for instance in collectd_plugin_ipmi_instances %}
+  <Instance "{{ instance }}">
+    {% if "host" in collectd_plugin_ipmi_instances[instance] %}
```
Comment (leifmadsen) [inline templates/ipmi.conf.j2:12]: Just curious, does this insert a bunch of extra whitespace around the parameters when the file is generated?

Comment (elfiesmelfie) [reply templates/ipmi.conf.j2:12]: The trailing whitespace should be stripped thanks to this change: https://github.com/infrawatch/collectd-config-ansible-role/commit/01d01f601410797686fa36a50faaf989f639cf03

I was getting line breaks disappearing and extra newlines being inserted when I adjusted any of the templates. Readability in the templates was effected by this too, so I added the whitespace management.

Comment (leifmadsen) [reply templates/ipmi.conf.j2:12]: Perfect thanks! I was wondering if you had implemented the ltrim_strip stuff somewhere.

**Review by leifmadsen — APPROVED** [review]
Looked through and didn't notice any typos or anything. Everything looks good to me.

**Pull request merged**